packages/puppeteer-core/src/cdp/PredefinedNetworkConditions.ts

```diff
@@ -30,13 +30,37 @@ import type {NetworkConditions} from './NetworkManager.js';
  */
 export const PredefinedNetworkConditions = Object.freeze({
```

Comment:
It would be helpful here to have a link to the DevTools source code in the comment here.